src/main/app/initialize.ts

```diff
@@ -198,10 +206,15 @@ function initializeBeforeAppReady() {
     refreshTrayImages(Config.trayIconTheme);
 
     // If there is already an instance, quit this one
-    const gotTheLock = app.requestSingleInstanceLock();
-    if (!gotTheLock) {
-        app.exit();
-        global.willAppQuit = true;
+    // eslint-disable-next-line no-undef
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    if (!__IS_MAC_APP_STORE__) {
```

Comment:
I assume this is because of https://github.com/electron/electron/issues/15958 but I don't see `LSMultipleInstancesProhibited` anywhere, so how do we handle multiple instances?

Reply:
Ahh, that's probably something I need to add. I will add that to `electron-builder.json`

Reply:
So I added that line and tested, but it doesn't seem to work as we want it to. You can still execute `./Mattermost.app/Contents/MacOS/Mattermost` from the command line and you'll get a second instance.

I'm not sure what a good way to handle this is.

Reply:
Oh yeah definitely you'll still get another instance if you launch the Electron binary from the command line. That's because it'll be a child process of the Terminal instance. But going through launchd using `open ./Mattermost.app` should behave more nicely.

Reply:
Looks like it doesn't actually matter if the line is there or not, launchd will always check if an instance is there first. I think that's something handled by the Mac App Store APIs, I feel like I read that somewhere at some point.

If that case is covered, are we okay to proceed? Normally the `singleInstanceLock` call would stop the command line binary from generating a second instance as well, but not sure if there's any concerns around that.

Reply:
Whoops actually `LSMultipleInstancesProhibited` is entirely different. From Apple docs: "A Boolean value indicating whether more than one user can launch the app simultaneously." We don't want that! I just saw it mentioned as a workaround in the Electron ticket, but didn't look into it further, sorry! 🙄